[2024-03-23T07:25:08.068+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: check_errors.check_value_for manual__2024-03-23T07:20:28.195572+00:00 [queued]>
[2024-03-23T07:25:08.081+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: check_errors.check_value_for manual__2024-03-23T07:20:28.195572+00:00 [queued]>
[2024-03-23T07:25:08.082+0000] {taskinstance.py:2193} INFO - Starting attempt 2 of 2
[2024-03-23T07:25:08.105+0000] {taskinstance.py:2217} INFO - Executing <Task(PythonOperator): check_value_for> on 2024-03-23 07:20:28.195572+00:00
[2024-03-23T07:25:08.116+0000] {standard_task_runner.py:60} INFO - Started process 1680 to run task
[2024-03-23T07:25:08.124+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'check_errors', 'check_value_for', 'manual__2024-03-23T07:20:28.195572+00:00', '--job-id', '240', '--raw', '--subdir', 'DAGS_FOLDER/check_erros.py', '--cfg-path', '/tmp/tmp7n6tc1kb']
[2024-03-23T07:25:08.129+0000] {standard_task_runner.py:88} INFO - Job 240: Subtask check_value_for
[2024-03-23T07:25:08.222+0000] {task_command.py:423} INFO - Running <TaskInstance: check_errors.check_value_for manual__2024-03-23T07:20:28.195572+00:00 [running]> on host ceb99a37e23c
[2024-03-23T07:25:08.348+0000] {taskinstance.py:2513} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='check_errors' AIRFLOW_CTX_TASK_ID='check_value_for' AIRFLOW_CTX_EXECUTION_DATE='2024-03-23T07:20:28.195572+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-03-23T07:20:28.195572+00:00'
[2024-03-23T07:25:08.353+0000] {logging_mixin.py:188} INFO - <Task(BigQueryCheckOperator): check_for_state_abb_table>
[2024-03-23T07:25:08.354+0000] {python.py:202} INFO - Done. Returned value was: <Task(BigQueryCheckOperator): check_for_state_abb_table>
[2024-03-23T07:25:08.364+0000] {xcom.py:664} ERROR - Object of type BigQueryCheckOperator is not JSON serializable. If you are using pickle instead of JSON for XCom, then you need to enable pickle support for XCom in your *** config or make sure to decorate your object with attr.
[2024-03-23T07:25:08.366+0000] {taskinstance.py:2731} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/utils/json.py", line 91, in default
    return serialize(o)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/serialization/serde.py", line 189, in serialize
    raise TypeError(f"cannot serialize object of type {cls}")
TypeError: cannot serialize object of type <class '***.providers.google.cloud.operators.bigquery.BigQueryCheckOperator'>

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskinstance.py", line 451, in _execute_task
    task_instance.xcom_push(key=XCOM_RETURN_KEY, value=xcom_value, session=session)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/utils/session.py", line 76, in wrapper
    return func(*args, **kwargs)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskinstance.py", line 3013, in xcom_push
    XCom.set(
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/utils/session.py", line 76, in wrapper
    return func(*args, **kwargs)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/xcom.py", line 247, in set
    value = cls.serialize_value(
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/xcom.py", line 662, in serialize_value
    return json.dumps(value, cls=XComEncoder).encode("UTF-8")
  File "/usr/local/lib/python3.8/json/__init__.py", line 234, in dumps
    return cls(
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/utils/json.py", line 104, in encode
    return super().encode(o)
  File "/usr/local/lib/python3.8/json/encoder.py", line 199, in encode
    chunks = self.iterencode(o, _one_shot=True)
  File "/usr/local/lib/python3.8/json/encoder.py", line 257, in iterencode
    return _iterencode(o, 0)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/utils/json.py", line 93, in default
    return super().default(o)
  File "/usr/local/lib/python3.8/json/encoder.py", line 179, in default
    raise TypeError(f'Object of type {o.__class__.__name__} '
TypeError: Object of type BigQueryCheckOperator is not JSON serializable
[2024-03-23T07:25:08.476+0000] {taskinstance.py:1149} INFO - Marking task as FAILED. dag_id=check_errors, task_id=check_value_for, execution_date=20240323T072028, start_date=20240323T072508, end_date=20240323T072508
[2024-03-23T07:25:08.495+0000] {standard_task_runner.py:107} ERROR - Failed to execute job 240 for task check_value_for (Object of type BigQueryCheckOperator is not JSON serializable; 1680)
[2024-03-23T07:25:08.544+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 1
[2024-03-23T07:25:08.583+0000] {taskinstance.py:3312} INFO - 0 downstream tasks scheduled from follow-on schedule check
